package.json

```diff
@@ -6,7 +6,7 @@
     "bugs": "http://issues.gpii.net/browse/GPII",
     "homepage": "http://gpii.net/",
     "dependencies": {
-        "universal": "GPII/universal#1a3434d970cb484956eb18310683bdfe473123d6"
+        "gpii-universal": "GPII/universal#36433633d01e59028ffefe0b4fcc59ef7d3886b2"
```

Comment:
You can use the last available version in EN PI EM, https://www.npmjs.com/package/gpii-universal :wink:

Reply:
In fact, you **must** use it :smile:

Reply:
well, okay if I **must** -- coming up.

Aside:  I based on your [change to windows master](https://github.com/GPII/windows/commit/bd537ded185c870399363df3e10b77f2c2be166d#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R15).  But, I see @amb26 [updated that just yesterday](https://github.com/GPII/windows/commit/5a2c6ebc6ebad653f39428176dc07975b4f2ae74#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L15).  Things change too fast here. ;-)